python/kfserving/docs/KFServingClient.md

```diff
@@ -16,11 +16,82 @@ The APIs for KFServingClient are as following:
 
 Class | Method |  Description
 ------------ | ------------- | -------------
+KFServingClient | [create_credentials](#create_credentials) | Create Credentials|
 KFServingClient | [create](#create) | Create KFService|
 KFServingClient | [get](#get)    | Get the specified KFService|
 KFServingClient | [patch](#patch)  | Patch the specified KFService|
 KFServingClient | [delete](#delete) | Delete the specified KFService |
 
+## create_credentials
+> create_credentials(storage_type, namespace=None, **kwargs)
```

Comment:
namespace="default"?

Reply:
@ellis-bigelow I perfer to None. If namespace is None, the code `utils.get_default_target_namespace()` will try to get current namespace if running in the cluster, otherwise will set to 'default'.